config-store-api/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>stargate</artifactId>
+		<groupId>io.stargate</groupId>
+		<version>0.0.18-SNAPSHOT</version>
```

Comment:
I guess this version may need to be adjusted... it does not find the parent module in my local run:

```
$ ./mvnw -f config-store-yaml/ test
[...]
[ERROR] Failed to execute goal on project config-store-yaml: Could not resolve dependencies for project io.stargate.config.store.yaml:config-store-yaml:jar:0.0.18-SNAPSHOT: Could not find artifact io.stargate.config-store:config-store-api:jar:0.0.18-SNAPSHOT -> [Help 1]

```

Reply:
I guess this is because the two new modules are not added to the parent pom, therefore they do not get published from the top-level `mvn install` command :thinking: